sql/src/main/java/org/apache/druid/sql/calcite/schema/SystemSchema.java

```diff
@@ -340,27 +332,6 @@ public TableType getJdbcTableType()
       return authorizedSegments.iterator();
     }
 
-    private CloseableIterator<DataSegment> getAuthorizedPublishedSegments(
```

Comment:
Is this authorization not needed anymore?

Reply:
Sorry, this authorization check is still required, because broker->coordinator call uses escalated client, so `MetadataResource` is using the escalated client. Will add this back.